File: app/src/main/java/com/admin/theapp/logger/ReleaseLogger.java
package com.admin.theapp.logger;

import android.support.annotation.NonNull;

import com.theapp.tools.Logger;

public class ReleaseLogger implements Logger {
    @Override
    public void d(@NonNull String tag, @NonNull String msg) { // default implementation ignored
    }

    @Override
    public void e(@NonNull String tag, @NonNull String msg) { // default implementation ignored
    }

    @Override
    public void e(@NonNull String tag, @NonNull String msg, @NonNull Throwable tr) { // default implementation ignored
    }

    @Override
    public void i(@NonNull String tag, @NonNull String msg) { // default implementation ignored
    }

    @Override
    public void v(@NonNull String tag, @NonNull String msg) { // default implementation ignored
    }

    @Override
    public void w(@NonNull String tag, @NonNull String msg) { // default implementation ignored
    }
}

File: app/src/main/java/com/admin/theapp/logger/ReleaseLogger.kt
package com.admin.theapp.logger

import com.theapp.tools.Logger

class ReleaseLogger : Logger {
    override fun d(tag: String, msg: String) { // default implementation ignored
    }

    override fun e(tag: String, msg: String) { // default implementation ignored
    }

    override fun e(tag: String, msg: String, tr: Throwable) { // default implementation ignored
    }

    override fun i(tag: String, msg: String) { // default implementation ignored
    }

    override fun v(tag: String, msg: String) { // default implementation ignored
    }

    override fun w(tag: String, msg: String) { // default implementation ignored
    }
}
